src/GAP.jl

```diff
@@ -118,6 +118,9 @@ function initialize(argv::Vector{String})
     # (which is created by `setup.jl`).
     append!(argv, ["--systemfile", abspath(@__DIR__, "..", "gap", "systemfile.g")])
 
+    # disable loading of all but the absolutely required packages
+    push!(argv, "-A")
```

Comment:
The addition of `-A` under the condition `haskey(ENV, "GAP_BARE_DEPS")` should then be removed from `__init__`